.github/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:latest
+COPY ./clash-rs/clash-x86_64-unknown-linux-musl /usr/bin/clash
```

Comment:
Will this fail when building arm image？

Reply:
The alpine base image exists as an arm build, which is reasonably fine. But in practice it needs to be tested on an arm machine, and I don't have one at the moment.

Reply:
Alpine is fine. I mean you have copied x86_64 clash-rs into arm image.

Reply:
You're right. I'll check.